### PR TITLE
 user-forgot-password

### PR DESCRIPTION
**Pull request by Ezejiohan**
Check this out

### CODE REVIEW

File: routes/users.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 const express = require('express');
-const { createUser } = require('../controllers/userC');
+const { createUser, userForgotPassword } = require('../controllers/userC');
 
 const userRoute = express.Router();
 
 userRoute.post('/register', (createUser));
+userRoute.post('/userForgotPassword', (userForgotPassword));
```
Comment (hizikex) [inline routes/users.js]:
```suggestion
userRoute.post('/forgot-password', (userForgotPassword));
```

You endpoint should not be camelCase.

File: controllers/userC.js [hunk 2]
```diff
@@ -23,4 +23,28 @@ const createUser = asyncWrapper(async(req, res, next) => {
...
+        subject: 'Forget password link',
+        message: message
+    });
+
+    res.status(200).json({
+        message: "Email has been sent"
+    });
+});
+
+module.exports = { createUser, userForgotPassword }
```
Comment (hizikex) [inline controllers/userC.js]: Please, Add an extra line to all your files.

**Review by hizikex — CHANGES_REQUESTED** [review]

File: routes/users.js (hunk 1 above)
Comment (Ezejiohan) [reply routes/users.js]: Okay, noted

File: controllers/userC.js (hunk 2 above)
Comment (Ezejiohan) [reply controllers/userC.js]: Okay, noted

**Pull request merged**